src/Controller/UI/ReportDatasetSubmission.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+namespace App\Controller\UI;
+
+use App\Entity\Dataset;
+use App\Entity\DatasetSubmission;
+use App\Entity\PersonDatasetSubmission;
+use App\Form\DatasetSubmissionType;
+use App\Form\PersonDatasetSubmissionType;
+use App\Repository\DatasetRepository;
+use Doctrine\Common\Collections\Collection;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * A controller for a Report of Research Groups and related Datasets.
+ *
+ * @return Response A Symfony Response instance.
+ */
+class ReportDatasetSubmission extends ReportController
+{
+    /**
+     * The default action for Dataset Review.
+     *
+     * @param string            $udi               A UDI.
+     * @param DatasetRepository $datasetRepository The dataset repository.
+     *
+     * @throws NotFoundHttpException When the dataset does not exist.
+     *
+     *
+     * @Route("/overview/{udi}")
+     *
+     * @return Response A Response instance.
+     */
+    public function overviewAction(string $udi, DatasetRepository $datasetRepository)
+    {
+        $dataset = $datasetRepository->findOneBy(['udi' => $udi]);
+
+        if (!$dataset instanceof Dataset) {
+            throw new NotFoundHttpException("Dataset $udi not found!");
+        }
+
+        $datasetSubmission = $dataset->getDatasetSubmission();
+
+        $data[] = array(
+            'label' => 'UDI',
+            'value' => $udi,
+        );
+
+        $fields = array(
+            'datasetContacts',
+            'authors',
+            'title',
+            'abstract',
+            'shortTitle',
+            'purpose',
+            'suppParams',
+            'suppMethods',
+            'suppInstruments',
+            'suppSampScalesRates',
+            'suppErrorAnalysis',
+            'suppProvenance',
+            'themeKeywords',
+            'placeKeywords',
+            'topicKeywords',
+            'spatialExtent',
+            'spatialExtentDescription',
+            'temporalExtentNilReasonType',
+            'temporalExtentDesc',
+            'temporalExtentBeginPosition',
+            'temporalExtentEndPosition',
+            'restrictions',
```

Comment:
The ticket's example csv shows that have Remotely hosted and cold storage attributes

Reply:
That's old.